Dockerfile.dev

```diff
@@ -1,9 +1,24 @@
+FROM golang:1.23 AS build
+
+ARG REACHABILITY_OVERRIDE_PUBLIC=false
+ARG BATCHFACTOR_OVERRIDE_PUBLIC=5
+
+WORKDIR /src
+# enable modules caching in separate layer
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . ./
```

Comment:
the latest `COPY` command includes the former as well, doesn't it? (so the at the latest COPY command could replace the former)